src/common/scheduler/policy/evaluation_result.go

```diff
@@ -0,0 +1,15 @@
+package policy
+
+//EvaluationResult is defined to carry the policy evaluated result.
+//
+//Filed 'Result' is optional.
+//Filed 'Error' is optional
+//
+type EvaluationResult struct {
```

Comment:
Let's not do this, Scheduler didn't even handle the Error in result.
Maybe the "Evaluate" can return a channel and an error to capture the error when generating the "evalchan" and other error can be sent via another channel,  *if needed*